windows/CodePushUnknownException.cs

```
@@ -0,0 +1,12 @@
+﻿using System;
```

Right... I forgot that Exceptions are not checked in C#.